docs/en/ingest-management/integrations/managed-integrations-content.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[managed-integrations-content]]
+= Managed integrations content
+
+++++
+<titleabbrev>Managed integrations content</titleabbrev>
+++++
+
+Most integration content installed by {fleet} isn’t editable. This content is tagged with a **Managed** badge in the {kib} UI. Managed content itself cannot be edited or deleted, however managed visualizations, dashboards, and saved searches can be cloned.   
+
+[role="screenshot"]
+image::images/system-managed.png[An image of the new managed badge.]
+
+With dashboards the cloned content is unlinked with the original reference configurations, so becomes independent from the original managed content. For example, with a cloned dashboard, the cloned panels become entirely self-contained copy without any dependencies on the original configuration. Clones can be customized and modified without accidentally affecting the original.
```

Comment:
Is this language about "reference configurations" standard in our docs? To me it seems more straightforward to say something like

"When a managed dashboard is cloned, any (library|linked|reference|something else) panels are made inline..."

This is just my opinion. I like your clarifications around what this actually means.

Also, is this a typo?

> the cloned panels become entirely self-contained copy

Reply:
I've re-worded the start of the paragraph to: